README.md

```diff
@@ -1,44 +1,6 @@
-# CIbyl
+CIbyl
```

Comment:
This is a markdown file but with rst syntax, will github be smart enough to parse it correctly? I just check and the link is not correctly shown. We can rename the file to rst or use markdown for it. Both would be fine IMO